Users/Docker_Image.rst

```diff
@@ -17,6 +17,10 @@ that they're running on and only requires applications being shipped
 with things not already running on the host computer. This gives a
 significant performance boost and reduces the size of the application.
 
+  .. note::
+
+    The coala Docker image does not support Python 2 analysis.
```

Comment:
Please move it to `coala as a Docker Image`, this has more relevance in that paragraph^^